### ISSUE TITLE
check for broken links in documentation

### ISSUE THREAD

**Issue by jairideout** [biocore/qiime#1897]

**Comment by jairideout** [comment]
Should probably take care of this after #1781. `pip install LinkChecker` seems to work pretty well.

**Comment by jairideout** [comment]
@ElDeveloper the [link to the data](http://bmf.colorado.edu/QIIME/QIIME_NM_2010.tgz) used in the original QIIME paper is down. Do you know where this file is hosted now?

**Comment by jairideout** [comment]
Also the [link](http://bmf.colorado.edu/QIIME/seed.16s.reference_model.sto.zip) in align_seqs.py pointing to a template Stockholm file for use with Infernal isn't working (this is another bmf.colorado.edu link).

**Comment by ElDeveloper** [comment]
Unfortunately I don't know what the location of any of these files is.
This pre-dates me joining the lab. Maybe @wasade or @antgonza would now.

**Comment by jairideout** [comment]
Thanks @ElDeveloper! I'm thinking of adding links to these two files, assuming that they'll be recovered and moved to ftp.microbio.me in the future. Are the following paths okay?

ftp://ftp.microbio.me/qiime/QIIME_NM_2010.tgz
ftp://ftp.microbio.me/qiime/seed.16s.reference_model.sto.zip

**Comment by ElDeveloper** [comment]
If the files can be recovered, then I can put them there for sure.

**Comment by jairideout** [comment]
Awesome, thanks! I've created a separate issue (#1914) to keep track of this.
